tests/integration-test.sh

```diff
@@ -18,6 +18,35 @@ set -x
 gsutil version -l
 set +x
 
+function run_tests() {
+    echo
+    echo "#######################################"
+    echo "# Running tests a first time          #"
+    echo "#######################################"
+    echo
+    pabot --verbose --processes $jobs --outputdir output $@
+    if [ $? -eq 0 ]; then
+        echo "All tests pass on first try"
+        return 0
+    fi
+    cp output/log.html  output/first_run_log.html
+    echo
+    echo "#######################################"
+    echo "# Running again the tests that failed #"
+    echo "#######################################"
+    echo
+    pabot --verbose --processes $jobs --outputdir output --rerunfailed output/output.xml --output rerun.xml $@
+    retval=$?
+    cp output/log.html  output/second_run_log.html
```

Comment:
This isn't the only log file we'd want to have two copies of; it'd be necessary to have two copies of the entire set of logs.